run/django/mysite/settings.py

```diff
@@ -63,7 +63,7 @@
 
 # SECURITY WARNING: don't run with debug turned on in production!
 # Change this to "False" when you are ready for production
```

Comment:
Could you move this warning up above to where it's set to `True` as well?

Reply:
Addressed your comment @dandhlee